src/CommonDBTM.php

```diff
@@ -6714,4 +6738,13 @@ public static function clearSearchOptionCache(): void
     {
         self::$search_options_cache = [];
     }
+
+    public static function getPostFormAction(string $form_action): ?string
+    {
+        return match ($form_action) {
+            'add', 'update' => 'back',
+            'delete', 'restore', 'purge' => 'list',
+            default => null,
+        };
```

Comment:
This code is here to be overriden in some cases. For now, only one generic-ed object overrides it, but since it's a child, I bet this will be overriden/updated specifically for children in other cases (like CommonDBChild for instance, which is for a future PR)

Reply:
When all forms will be handled, we will see in the future if it is really needed or if we are able to find an automatic way to define the expected behaviour.